shared/src/simulation/activities/transfer-vehicles.ts

```diff
@@ -0,0 +1,193 @@
+import { Type } from 'class-transformer';
+import { IsString, IsUUID, ValidateNested } from 'class-validator';
+import { groupBy } from 'lodash-es';
+import {
+    MissingTransferConnectionRadiogram,
+    RadiogramUnpublishedStatus,
+} from '../../models/radiogram';
+import { publishRadiogram } from '../../models/radiogram/radiogram-helpers-mutable';
+import {
+    currentSimulatedRegionIdOf,
+    getCreate,
+    isInSimulatedRegion,
+    isInSpecificSimulatedRegion,
+    TransferStartPoint,
+} from '../../models/utils';
+import { VehicleResource } from '../../models/utils/vehicle-resource';
+import { TransferActionReducers } from '../../store/action-reducers/transfer';
+import { getElement } from '../../store/action-reducers/utils';
+import { cloneDeepMutable, UUID, uuidValidationOptions } from '../../utils';
+import { IsValue } from '../../utils/validators';
+import {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    ExerciseSimulationEvent,
```

Comment:
Is there some reason for this unused import? Or did you just use the wrong auto-fix (eslint ignore instead of remove import)?

Reply:
Unintentional I will remove it

Reply:
Done